README.md

```diff
@@ -1,71 +1,43 @@
-Terraform Provider
-==================
+# To update Terrform-provider-chef
 
-- Website: https://www.terraform.io
-- [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
-- Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
+## Requirement
 
-<img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" width="600px">
+Install gpg, go, sha256sum, goreleaser if you don't have them yet:
 
-Requirements
-------------
-
--	[Terraform](https://www.terraform.io/downloads.html) 0.10.x
--	[Go](https://golang.org/doc/install) 1.11 (to build the provider plugin)
-
-Building The Provider
----------------------
-
-Clone repository to: `$GOPATH/src/github.com/terraform-providers/terraform-provider-chef`
-
-```sh
-$ mkdir -p $GOPATH/src/github.com/terraform-providers; cd $GOPATH/src/github.com/terraform-providers
-$ git clone git@github.com:terraform-providers/terraform-provider-chef
+```shell
+brew install gpg
+brew install go
+brew install goreleaser
+brew install coreutils # contains sha256sum
 ```
 
-Enter the provider directory and build the provider
+## To publish
 
-```sh
-$ cd $GOPATH/src/github.com/terraform-providers/terraform-provider-chef
-$ make build
-```
+you'll need the new version number, a GPG_FINGERPRINT:
```

Comment:
Should make it clear that you will have actually had to generate a key beforehand